CODE_OF_CONDUCT.md

```diff
@@ -17,23 +17,23 @@ diverse, inclusive, and healthy community.
 Examples of behavior that contributes to a positive environment for our
 community include:
 
-* Demonstrating empathy and kindness toward other people
-* Being respectful of differing opinions, viewpoints, and experiences
-* Giving and gracefully accepting constructive feedback
-* Accepting responsibility and apologizing to those affected by our mistakes,
+- Demonstrating empathy and kindness toward other people
+- Being respectful of differing opinions, viewpoints, and experiences
+- Giving and gracefully accepting constructive feedback
+- Accepting responsibility and apologizing to those affected by our mistakes,
```

Comment:
It's a nit but I prefer `*` for bullet lists over `-`.

Reply:
Markdownlint supports the asterix style: https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md004---unordered-list-style

Reply:
Prettier does not support it.

Reply:
I'd rather drop prettier than change our formatting just to cater to prettier.

Reply:
Doesn't pre-commit download and install node behind the scenes?

Reply:
Yes, it does but I do not think it is a burden on the developer experience.

Reply:
Having to install pre-commit is an additional step and thereby somewhat of a burden. Personally I rather avoid tools that download stuff behind the scenes.

Reply:
And the drawbacks of depending on an entire other ecosystem don't go away just because you install it automatically.

Reply:
Yeah I've given it some thought... and I think I'd prefer not to add Prettier as a dev dependency. I'm happy to run Prettier as a one-time thing over the YAML files in this PR, and also to clean up the Markdown a bit (you fixed some good things in the README like trailing space, but we could roll back the `*` changes and such), but I'd rather not add Prettier on an ongoing basis.